### ISSUE TITLE
[Request]: Adding/editing ALT text while editing posts

### ISSUE THREAD

**Issue by LukaszH77** [GeopJr/Tuba#319]
### Describe the request

It would be useful to be able to add the missing alt text when editing an entry or to change the already added one.

### Implementation Details

- [ ] This should be an option in settings.
- [ ] This should be only available to some fediverse backends. (Include which ones on the above field).
- [ ] This is client-only (and shouldn't sync with the instance).
- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by GeopJr** [comment]
Uh, undocumented, I'll update mastodon's docs
